src/frontend/src/helpers/Storage/WebStorage.ts

```diff
@@ -417,7 +420,7 @@ class WebStorage extends AbstractStorage implements AbstractWebStorage {
     }
     const changesSent = await this.storage.getChangeLogs();
     const settingsSent = await this.storage.getSettings();
-    if (!fetchServerChanges) {
+    if (! fetchServerChanges) {
       if (changesSent.length === 0) {
```

Comment:
We should also test if we've made any changes to project settings, but that can wait.  The easiest way to do this is to have a localStorage variable that's true if and only if we've made any changes.